### DISCUSSION TITLE
Update your github-user-contrubutions!

### DISCUSSION THREAD

**Discussion by ankitsumitg** [ankitsumitg/github-user-contributions#1] [Announcements]
## 👋 Welcome!
Comment in this thread to update your contributions!
- Format: ` @username `

**Comment by ankitsumitg** [comment]
Adding new contributors -> This should `PASS`
- @ankitsumitg
- @sumitankitg130
- @canering
- @Luj8n
- @mukeshprasad
- @davidtrovisco
- @haydencordeiro

**Comment by ankitsumitg** [comment]
This should `FAIL` , Since no user are present in this comment

**Comment by ankitsumitg** [comment]
Adding 1 more to test final update @jorgebucaran

**Comment by ankitsumitg** [comment]
This should `FAIL`, since user does not exist
@idonotexisttttttttttttttttt

**Comment by ankitsumitg** [comment]
New contributor @daniijal

**Comment by MattTheTekie** [comment]
@MattTheTekie

**Comment by JamieCurnow** [comment]
@JamieCurnow

**Comment by pugson** [comment]
@pugson
